frontend/javascripts/components/slider.tsx

```diff
@@ -0,0 +1,100 @@
+import { Slider as AntdSlider, type SliderSingleProps } from "antd";
+import type { SliderRangeProps } from "antd/lib/slider";
+import type { WheelEventHandler } from "react";
+
+const DEFAULT_WHEEL_FACTOR = 0.02;
+const DEFAULT_STEP = 1;
+
+type SliderProps = (SliderSingleProps | SliderRangeProps) & {
+  wheelFactor?: number;
+  disableOnWheel?: boolean;
+};
+
+const getDiffPerSliderStep = (
+  sliderRange: number,
+  factor: number | undefined = DEFAULT_WHEEL_FACTOR,
+  step: number | undefined | null,
+) => {
+  let stepNotNull = step || DEFAULT_STEP;
+  let result = (factor || DEFAULT_WHEEL_FACTOR) * sliderRange;
+  if (result < stepNotNull) return stepNotNull;
+  return result;
```

Comment:
Good point. I also tried that. The problem I see with the default parameter is that `step` (coming from the slider props) can be initialized as  `undefined`. And in that case, the default param would not be set, leading to a typing error when comparing `result < stepNotNull`.
You are right for the wheelFactor though!

Reply:
`step` is a property from a slider that is passed through, thats why I named it that way. Your are right that the returned number of that function cant be smaller than `step`, but its not really a `minStep` because its not the minimum slider step. Does that make sense?

Reply:
sounds like a very good idea!